API/Models/Travel/TravelRepository.cs

```diff
@@ -4,48 +4,27 @@
 using System.Runtime.CompilerServices;
 using WebApplication2.Utilities;
 using System.Text.RegularExpressions;
+using Microsoft.EntityFrameworkCore;
+using WebApplication2.Data;
 using FileIO = System.IO.File;
 
 namespace WebApplication2.Models.Travel;
 public class TravelRepository : ITravelRepository
 {
-    private List<TravelModel> TravelList;
+    private DbSet<TravelModel> TravelList;
+    public DataContext _context;
     public const string _jsonPath = "TravelDB.json";
-    public TravelRepository()
+    public TravelRepository(DataContext context)
```

Comment:
Nėra pats geriausias sprendimas dėti funkcijas tokias į objekto konstruktorių, nes ne visi reikiami duomenys dar gali būti paruošti, kol konstruktorius kuriasi. Būt geriau sukurt objektą ir tada funkciją ant to objekto iškviest.